services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_utils.py

```diff
@@ -151,7 +152,7 @@ async def service_free_reserved_disk_space(
     scheduler_data: SchedulerData = _get_scheduler_data(app, node_id)
     try:
         await sidecars_client.free_reserved_disk_space(scheduler_data.endpoint)
-    except BaseHttpClientError as e:
+    except (BaseHttpClientError, TaskExceptionError) as e:
```

Comment:
same here?
I find it correct that the client does it as you do it here. but the underlying tool should not as well.
the whole traceback should be printed here actually. via log.exception.

Reply:
Please refer to my comment above